constants/links.ts

```diff
@@ -38,4 +38,9 @@ const SocialMedia: { [key: string]: TextLink } = {
   },
 } as const;
 
-export { LearnStartLearningUrl, SocialMedia };
+const EcosystemJoinLink: TextLink = {
+  url: "https://github.com/qiskit-community/ecosystem#ecosystem--",
+  label: "Join the ecosystem",
+};
```

Comment:
This definition should be in the **pages/ecosystem.vue** file, since it's a link that is only used there.

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3443/commits/83cbb3b3030bb36070a9ad5860b1b8d55b12cfe7